clients/src/test/java/org/apache/kafka/clients/admin/AdminClientUnitTestEnv.java

```diff
@@ -113,7 +114,7 @@ public MockClient kafkaClient() {
 
     @Override
     public void close() {
-        this.adminClient.close();
+        this.adminClient.close(Duration.ZERO);
```

Comment:
The default implementation of `close()` is to wait forever. What's the reasoning for using `Duration.ZERO` here?

Reply:
Indefinite blocking seems like a bad combination when time is often mocked.

Reply:
That said, I don't think I actually need this change for this patch, so I can revert it.